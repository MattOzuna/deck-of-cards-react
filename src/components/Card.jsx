import { useState } from "react";

const Card = ({value, suit, image}) => {

  return <div>
      <img src={image} alt={`${value} of ${suit}`} />
    </div>;
};

export default Card;
