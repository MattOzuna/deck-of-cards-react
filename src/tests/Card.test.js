import { render } from "@testing-library/react";
import Card from "../components/Card";

it("should render", () => {
  render(<Card />);
});

it("should match snapshot", () => {
  const { asFragment } = render(
    <Card
      image='img'
      value='6'
      suit="Hearts"
      key='1'
      idx='1'
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
