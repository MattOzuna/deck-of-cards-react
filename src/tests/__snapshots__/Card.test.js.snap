// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
    style="z-index: 1;"
  >
    <img
      alt="6 of Hearts"
      src="img"
    />
  </div>
</DocumentFragment>
`;
